app/views/helpers/index/normal/entry_bottom.phtml

```diff
@@ -38,22 +38,43 @@
 	// @phpstan-ignore-next-line
 	if ($bottomline_labels) {
 	?><li class="item labels">
-		<div class="item-element dropdown dynamictags">
-			<div id="dropdown-labels-<?= $this->entry->id() ?>" class="dropdown-target"></div>
-			<a class="dropdown-toggle" href="#dropdown-labels-<?= $this->entry->id() ?>">
-				<?= _i('label') ?><?= _t('index.menu.tags') ?>
-			</a>
-			<ul class="dropdown-menu dropdown-menu-scrollable scrollbar-thin">
-				<li class="dropdown-header">
-					<?= _t('index.menu.tags') ?>
-					<?php if (FreshRSS_Auth::hasAccess()) { ?>
-						<a href="<?= _url('tag', 'index') ?>"><?= _i('configure') ?></a>
-					<?php } ?>
-				</li>
-				<!-- Ajax -->
-			</ul>
-			<a class="dropdown-close" href="#close">❌</a>
-		</div>
+		<?php
+		$showDropdown = true;
+		if (!FreshRSS_Auth::hasAccess()) {
+			$tagDAO = FreshRSS_Factory::createTagDao();
+			$tagsForEntry = $tagDAO->getTagsForEntry($this->entry->id(), true);
```

Comment:
I am worried of the performance penalty of that. Any reason for not using the current method also in anonymous mode?